charts/coredns/coredns-pod-delete/engine.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: engine-coredns
+  namespace: kube-system
+spec:
+  appinfo:
+    appns: kube-system
```

Comment:
Can you please add single quotes also for  `appns`, `appkind` and `jobCleanUpPolicy` in all files